packages/good-design/src/apps/goodid/controllers/SegmentationController.tsx

```diff
@@ -44,5 +50,14 @@ export const SegmentationController = () => {
     }
   };
 
-  return <SegmentationWizard onDone={onDone} onLocationRequest={onLocationRequest} />;
+  return (
+    <SegmentationWizard
+      onDone={onDone}
```

Comment:
ondone shouldnt be hardcoded

Reply:
That got fixed in the checkAVailableOffers PR: https://github.com/GoodDollar/GoodWeb3-Mono/blob/49a5a2036841d56274074729b7e65fbafa9afa93/packages/good-design/src/apps/goodid/controllers/SegmentationController.tsx#L28